README.md

```diff
@@ -47,6 +47,7 @@ disk_additional_disks:
 
 You can add:
 * `disk_package_use` is the required package manager module to use (yum, apt, etc). The default 'auto' will use existing facts or try to autodetect it.
+* `disk_enable_partioning: false` to optionnaly disable partitioning
```

Comment:
You can drop `optionally` and just say `to disable partitioning` (or update the typo `optionnaly -> optionally`)